spec/models/application_proceeding_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe ApplicationProceedingType, type: :model do
+  subject { described_class.new }
+
+  let!(:applicationProceedingType) { ApplicationProceedingType.create }
+
+  it 'should belong to an proceeding_type and legal_aid_application' do
+    expect(subject.legal_aid_application_id).to eq(applicationProceedingType.id)
```

Comment:
😕 Are we sure this test is doing something useful?

```ApplicationProceedingType.create``` 

seems to just be creating an empty record so this assertion:

```ruby
expect(subject.legal_aid_application_id).to eq(applicationProceedingType.id)
```

is probably just passing because both sides of the expectation are `nil`

what you probably would like to assert IMO is:

* create a legal aid application record
* create a proceeding type record
* create an application proceeding type record that takes as arguments the two models created in the previous steps
* now you can make the assertion where you check that 

```ruby
expect(application_proceeding_type.legal_aid_application).to eq(legal_aid_application)
expect(application_proceeding_type.proceeding_type).to eq(proceeding_type)
```